pkg/services/multildap/multildap_test.go

```diff
@@ -446,3 +446,76 @@ func TestMultiLDAP(t *testing.T) {
 		})
 	})
 }
+
+// MockLDAP represents testing struct for ldap testing
+type MockLDAP struct {
```

Comment:
Nit, but do you mind not exporting `MockLDAP`?
```suggestion
type mockLDAP struct {
```

Reply:
It's not exposed since it's in a test file

Reply:
@marefr I know, but there's no reason to export it in a test build either, just as a matter of code hygiene. It's best to avoid exporting symbols (by capitalizing the first letter), unless they actually should be.

Reply:
Fixed